nomos/models/tool.py

```diff
@@ -213,6 +219,50 @@ def rename_pydantic_model(
         except Exception as e:
             raise ValueError(f"Could not load CrewAI tool {tool_id}: {e}")
 
+    @classmethod
+    def from_mcp_server(cls, server: "MCPServer") -> List["Tool"]:
+        """
+        Create a Tool instance from a MCP server.
+
+        :param server: The MCP server instance.
+        :return: A list of Tool instances.
+        """
+        mcp_tools = server.get_tools()
+        tools = []
+        for mcp_tool in mcp_tools:
+            tool_name = f"{server.id}/{mcp_tool.name}"
+            tool = cls(
+                name=tool_name,
+                description=mcp_tool.description,
+                function=lambda name=mcp_tool.name, **kwargs: cls._run_mcp_tool(
```

Comment:
Do we have to pass the name here. and instead of having cls._run_mcp_tool, use server.run_tool(name=mcptool.name, **kwargs)
```suggestion
                function=lambda name=mcp_tool.name, **kwargs: server.run_tool(name=mcptool.name, **kwargs)
```